src/hosted_db/sqlite.rs

```diff
@@ -36,3 +47,35 @@ pub fn run_sqlite_query(path: &PathBuf, query: &str) -> Result<QueryResult, AybE
         rows: results,
     })
 }
+
+pub async fn run_sqlite_query(
```

Comment:
Consider this function name relative to the other one in this file: can they be more distinct?

Reply:
Done!